### PR TITLE
Extended Euclidean Algorithm in PHP

### LINKED ISSUE #2906: Extended Euclidean Algorithm
- [x] C
- [x] C++
- [x] C#
- [x] CoffeeScript
- [ ] Dart - @Jitmanew-Tyagi
- [ ] Go
- [x] Java - @lakshyajit165 
- [x] JS - @Satyam3976
- [ ] Kotlin - @Manish-cloud
- [x] Php - @utkarshanehe
- [x] Python
- [ ] Readme - @Amitsharma45
- [x] Ruby - @atarax665
- [ ] Rust
- [x] TypeScript - @shivanidalmia1203

### PR DESCRIPTION
**Pull request by utkarshanehe**
Fixes #2906

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Program for Extended Euclidean Algorithm in PHP

#### Languages Used:
- PHP

#### Files Added:
- Extended_Euclidean_Algorithm.php

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by shivamgupta57121** [comment]
You have to complete the checklist before making pull request.

File: Extended_Euclidean_Algorithm/Extended_Euclidean_Algorithm.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    $gcd = gcd($b % $a, $a, $x, $y); 
+
+    // Update x and y using results of recursive call 
+    $x = $y - ($b / $a) * $x; 
+    $y = $x; 
+    return $gcd; 
+} 
+
+// Driver Code 
+$x = 0;
```
Comment (raghav-dalmia) [inline Extended_Euclidean_Algorithm/Extended_Euclidean_Algorithm.php:23]: take input dynamically

File: Extended_Euclidean_Algorithm/Extended_Euclidean_Algorithm.php [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+// Driver Code 
+$x = 0; 
+$y = 0; 
+$a = 20; 
+$b = 5; 
+echo "\nGCD of ", $a, " and ", $b, " is ", gcd($a, $b, $x, $y); 
+
+/*
+    Input:
```
Comment (raghav-dalmia) [inline Extended_Euclidean_Algorithm/Extended_Euclidean_Algorithm.php:30]: These lines should be left indent

File: Extended_Euclidean_Algorithm/Extended_Euclidean_Algorithm.php [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    $y = $x; 
+    return $gcd; 
+} 
+
+// Driver Code 
+$x = 0; 
+$y = 0; 
+$a = 20; 
+$b = 5; 
+echo "\nGCD of ", $a, " and ", $b, " is ", gcd($a, $b, $x, $y);
```
Comment (raghav-dalmia) [inline Extended_Euclidean_Algorithm/Extended_Euclidean_Algorithm.php:27]: also print `x` and `y`

File: Extended_Euclidean_Algorithm/Extended_Euclidean_Algorithm.php [hunk 4]
```diff
@@ -0,0 +1,35 @@
+<?php 
+//Function to find GCD
```
Comment (raghav-dalmia) [inline Extended_Euclidean_Algorithm/Extended_Euclidean_Algorithm.php:2]: add a empty line after `<?php `

**Comment by utkarshanehe** [comment]
@raghav-dalmia @jainaman224  @asha15 
I have implemented all the suggested changes. Please review my PR.

**Pull request merged**